git_test.go

```diff
@@ -14,10 +14,10 @@ var _ = Describe("Git", func() {
 	path := os.Getenv("PATH")
 
 	BeforeEach(func() {
-		os.Setenv("PATH", fmt.Sprintf("./fixtures:%s", path))
+		_ = os.Setenv("PATH", fmt.Sprintf("./fixtures:%s", path))
```

Comment:
@Chadius why this change? I was looking around the internet for whether `os.Setenv` should return anything and I see examples of both.